src/tests/ftest/util/server_utils.py

```diff
@@ -290,3 +271,31 @@ def kill_server(hosts):
     for host in hosts:
         subprocess.call("ssh {0} \"{1}\"".format(host, '; '.join(kill_cmds)),
                         shell=True)
+
+def storage_prepare(hosts):
+    """
+    Prepare the storage on servers using the DAOS server's yaml settings file.
+
+    Args:
+        hosts (str): a string of comma-separated host names
+    """
+    try:
+        cmd = ("sudo /usr/bin/daos_server storage prep-nvme "
+               "--target-user=\"{0}\" --hugepages=4096"
+               .format(getpass.getuser()))
+        clustershell_execute(cmd, hosts, timeout=120)
+    except ClusterCommandFailed as error:
+        raise ServerFailed(error)
```

Comment:
I might add some context here, along the lines of:

```suggestion
        raise ServerFailed("Error preparing NVMe storage:\n{}".format(error))
```